src/functions/indAffine.jl

```diff
@@ -26,11 +26,11 @@ immutable IndAffine{T <: RealOrComplex} <: IndicatorConvex
   end
 end
 
-IndAffine{T <: RealOrComplex}(A::AbstractArray{T,2}, b::AbstractArray{T,1}) =
-  IndAffine{T}(A, b)
+IndAffine{T <: RealOrComplex, M<:AbstractArray{T,2}, V<:AbstractArray{T,1}}(A::M, b::V) =
+  IndAffine{T,M,V}(A, b)
```

Comment:
No I think you are right, but then we should be able to get rid of several of the inner constructors right? But maybe we want to keep them for some reason?